qualification/__init__.py

```diff
@@ -604,9 +615,11 @@ def chunk_place(data_ptr, data_size, user_data_ptr):
         use_ibv,
         stream_config,
         max_chunks,
+        beam_ants_dtype.itemsize,
         chunk_place.ctypes,
         lambda stream: katgpucbf.recv.Chunk(
             present=np.empty((n_beams, n_substreams), np.uint8),
+            extra=np.zeros((n_beams, n_substreams), beam_ants_dtype),
```

Comment:
Admittedly haven't been in the mindset of qualification (and katsdpcontroller) for some time; not entirely reconciling the idea that this Receiver is receiving data **output by the XBEngine**. More specifically,
* Why is it that the shape is `(n_beams, n_substreams)`?
* e.g. In the BSend, each beam data product is referred to as a substream
* I'm likely missing something in the change in naming convention/environment from `katgpucbf/src/` to here.

Reply:
There is typically a corner turn involved: each XBEngine sends part of the band for every beam, while a typical receiver will consume the entire band for a single beam. So the sender's substreams are not the same as the receiver's substreams. In this case we're consuming the whole band for multiple beams, so I guess more properly one should call this `n_substreams_per_beam` or something, but that's quite a mouthful.